addons/website/static/src/interactions/video/media_video.js

```diff
@@ -55,7 +56,7 @@ export class MediaVideo extends Interaction {
         // 'data-oe-expression' one (the latter is used as a workaround in 10.0
         // system but should obviously be reviewed in master).
 
-        let src = escape(this.el.getAttribute("data-oe-expression") || this.el.getAttribute("data-src"));
```

Comment:
Note: please someone double check this.
It seems like a really bad practice to escape something before knowing where it will be used.
It is either useless or unreliable.
The escaping should be done in the code that uses this value in an unsafe place (if there is any).